src/nft/css/sprinkles.css.ts

```diff
@@ -216,7 +216,9 @@ export const vars = createGlobalTheme(':root', {
     '12': '12px',
     '14': '14px',
     '16': '16px',
+    '18': '18px',
     '20': '20px',
+    '22': '22px',
```

Comment:
minor nit: since these two border radii are only used once, can you add them as a custom `style={{...}}` for their respective elements instead of adding them to sprinkles?

Reply:
sg removed